site/app/controllers/grading/ElectronicGraderController.php

```diff
@@ -3335,6 +3362,7 @@ private function getMarkStats(Mark $mark, User $grader) {
      */
     private function getStats(Gradeable $gradeable, User $grader, bool $full_stats, &$total_graded, &$total_total) {
         $num_components = $this->core->getQueries()->getTotalComponentCount($gradeable->getId());
+        //$verified_components = $this->core->getQueries()->getTotalVerifiedComponentCount($gradeable->getId());
```

Comment:
can you remove this line and every similar unnecessary comment in this PR. Also can you make sure your package isn't different then the one from main